src/lms-ds-loader/edfi_lms_ds_loader/db_operations/MSSQL_sql_builder.py

```diff
@@ -187,38 +187,45 @@ def insert_new_records_to_production_for_assignment_and_user_relation(
 
 
 def insert_new_records_to_production_for_attendance_events(
-        table: str,
         insert_columns: str,
         select_columns: str) -> str:
     return f"""
 INSERT INTO
-    lms.{table}
+    lms.LMSUserAttendanceEvent
```

Comment:
This change seems fine... 

Do you remember why we made this a template before, instead of hard-coding the name as you have now done?